packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -61,22 +63,38 @@ export class MeterProvider implements api.MeterProvider {
     return this._meters.get(key)!;
   }
 
-  shutdown(cb: () => void = () => {}): void {
-    this._shutdownAllMeters().then(() => {
-      setTimeout(cb, 0);
-    });
+  shutdown(): Promise<void> {
```

Comment:
async?

Reply:
it is not needed as it return already a function which is promise